src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -28,6 +33,12 @@
       rightJoystick.getRoll()
   );
 
+  private final Grabber grabber = new Grabber(
```

Comment:
Good place to put this. You can make this all one line. We had only split `drive` across multiple lines because we couldn't get it all to fit in under 100 characters.

```java
private final Grabber grabber = new Grabber(IO.grabSolenoid, IO.armSolenoid);
```